packages/core/stories/combo-box/combo-box.stories.tsx

```diff
@@ -653,6 +655,54 @@ MultiplePillsTruncated.args = {
   truncate: true,
 };
 
+export const SecondaryLabel: StoryFn<ComboBoxProps<Person>> = (args) => {
+  const [value, setValue] = useState(args.defaultValue?.toString() ?? "");
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    // React 16 backwards compatibility
+    event.persist();
+
+    const value = event.target.value;
+    setValue(value);
+  };
+
+  const handleSelectionChange = (
+    event: SyntheticEvent,
+    newSelected: Person[]
+  ) => {
+    if (newSelected.length === 1) {
+      setValue(newSelected[0].firstName);
+    } else {
+      setValue("");
+    }
+  };
+
+  const options = people.filter(
+    (person) =>
+      person.firstName.toLowerCase().includes(value.trim().toLowerCase()) ||
+      person.lastName.toLowerCase().includes(value.trim().toLowerCase())
+  );
+
+  return (
+    <ComboBox<Person>
+      {...args}
+      onChange={handleChange}
+      onSelectionChange={handleSelectionChange}
+      value={value}
+      valueToString={(person) => person.displayName}
+    >
+      {options.map((person) => (
+        <Option key={person.id} value={person}>
+          <StackLayout gap={0.25} align="start">
+            <Text variant="primary">{person.firstName}</Text>
+            <Label variant="secondary">{person.lastName}</Label>
```

Comment:
Label isn't semantically correct here we could probably just use:

```suggestion
            <Text variant="secondary">{person.lastName}</Text>
```

But we should also use the color prop